src/main/java/views/session_load/SessionLoadMenuV.java

```diff
@@ -0,0 +1,84 @@
+package views.session_load;
+
+import use_cases.session_load.SessionLoadException;
+import use_cases.session_load.SessionLoadOD;
+import views.main_menu.MainMenuV;
+
+import javax.swing.*;
+import java.awt.*;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+
+public class SessionLoadMenuV extends JPanel implements SessionLoadMenuVB, ActionListener {
+    private final SessionLoadC controller;
+    private final MainMenuV mainMenuV;
+
+
+    public SessionLoadMenuV(SessionLoadC controller, MainMenuV mainMenuV) {
+        this.controller = controller;
+        this.mainMenuV = mainMenuV;
+
+        JLabel title = new JLabel("Load session");
+        title.setAlignmentX(Component.CENTER_ALIGNMENT);
+
+        JPanel buttons = new JPanel();
+        JButton load_new = new JButton("Create new session");
+        JButton load_file = new JButton("Load session from a file");
+
+        load_new.addActionListener(this);
+        load_file.addActionListener(this);
```

Comment:
Although you used ActionListener, you did not implement the actionPerformed method from ActionListener interface.

Reply:
It is implemented further below

Reply:
Yeah, sorry, I didn't see it.